cpg-analysis/src/main/kotlin/de/fraunhofer/aisec/cpg/query/FlowQueries.kt

```diff
@@ -265,8 +265,13 @@ fun dataFlowWithValidator(
  * computing this node as the result of a function.
  */
 data class NodeWithAssumption(val node: Node) : HasAssumptions {
+    /** See [HasAssumptions.assumptions]. */
```

Comment:
Isn't there a link by default to the parent implementation's documentation? -> can be left out